e2e-tests/web/test/specs/navigateToSearchResults.e2e.ts

```diff
@@ -0,0 +1,29 @@
+import { URL } from 'url'
+import { contentSearch, defaultCity, filter, language, Routes } from '../../../shared/constants'
+import { navigateToDashboard } from '../utils/navigationUtils'
+import DashboardPage from '../pageobjects/dashboard.page'
+import SearchPage from '../pageobjects/search.page'
+
+describe('navigateToSearchResult', () => {
+  it('should open and search content', async () => {
+    const searchPath = Routes.search
+    await navigateToDashboard()
+
+    const searchIcon = await DashboardPage.searchIcon
+    await searchIcon.click()
+
+    const searchUrl = await browser.getUrl()
+    const parsedSearchUrl = new URL(searchUrl)
+    expect(parsedSearchUrl.pathname).toContain(searchPath)
+
+    const searchBar = await SearchPage.search
+    expect(searchBar).toBeDefined()
+    await browser.keys(contentSearch)
+
+    const firstResult = await $('=Language')
+    const secondResult = await $('=Language courses')
+
+    expect(firstResult).toBeDefined()
+    expect(secondResult).toBeDefined()
```

Comment:
I think it would be nice if we could open a search result. Also you could assert that the url contains the query.

Reply:
I added opening a search result. 
I also thought about the query, but decided not to add this, as I assumed e2e-tests are for main user-stories. If the url contains the query is more something for a unit test, isn't it?